### PR TITLE
Add May event at Hello Customer

### PR DESCRIPTION
**Pull request by tomkerkhove**

### CODE REVIEW

File: _posts/events/2017-05-03---azure-machine-learning-and-securing-azure-assets.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+Hello Customer is happy to host an event in the Ghelacomo arena to talk about finding nuggets with Azure Machine Learning (ML) Studio and Mike Marting will talk about how and where azure security works and what you can do to secure your Azure resources.
+
+## Sessions
+### Finding nuggets with Azure ML Studio
+Currently there are not a lot of point-and-click data science tools, Azure ML Studio is among the first with a very specific offering. During this session we will make a text analysis exercise on azure ml studio and see where the pitfalls are. After the exercise we will discuss the advantages and disadvantages of using Azure ML Studio.
+
+<img src="/assets/media/speakers/tijl-carpels.png" alt="Tijl Carpels" align="left" width="100" height="100" style="margin-right: 20px;">**Speaker:** *As a Data Scientist, Tijl Carpels is designing and implementing Natural Language Processing pipelines at Hello Customer. His interests range from distributed computing to machine learning and developing chatbots.*
+
+### Enter The Matrix: Securing Azure's Assets
```
Comment (tomkerkhove) [inline _posts/events/2017-05-03---azure-machine-learning-and-securing-azure-assets.md]: This one looks funky but I'm not a web expert. Feel free to let me know how I can fix it

File: _posts/events/2017-05-03---azure-machine-learning-and-securing-azure-assets.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+Ghelamco Arena – Meetdistrict (Hosted by **Hello Customer**)<br/>
+Ottergemsesteenweg Zuid 808<br />
+9000 Gent<br />
+Belgium
+
+*The event takes place on the 1st floor and is accessible via the entrance next to Albert Hein.<br />
+[Here's how you can drive to Hello Customer.](../../assets/media/documents/Driving-To-Hello-Customer.pdf)*
+
+## Register via EventBrite
+No registrations available yet.
```
Comment (tomkerkhove) [inline _posts/events/2017-05-03---azure-machine-learning-and-securing-azure-assets.md]: If you have this information available I can add it for you.

**Comment by maartenba** [comment]
@tomkerkhove Is this one good to merge? (will add Eventbrte afterwards)

**Comment by tomkerkhove** [comment]
:shipit:

**Pull request merged**